pages/linux/mdadm.md

```diff
@@ -4,24 +4,24 @@
 
 - Create array:
 
-`mdadm --create {{/path/to/raid_device_file}} --level {{raid_level}} --raid-devices {{number_of_disks}} {{/path/to/disk_device_file}}`
+`mdadm --create {{path/to/raid_device_file}} --level {{raid_level}} --raid-devices {{number_of_disks}} {{path/to/disk_device_file}}`
 
 - Stop array:
 
-`mdadm -S {{/path/to/raid_device_file}}`
+`mdadm -S {{path/to/raid_device_file}}`
 
 - Mark disk as failed:
 
-`mdadm {{/path/to/raid_device_file}} -f {{/path/to/disk_device_file}}`
+`mdadm {{path/to/raid_device_file}} -f {{path/to/disk_device_file}}`
```

Comment:
The paths for this command is not good at all. Because this command use partitions so, I think this should be the placeholders:
```suggestion
`mdadm {{/dev/md0}} -f {{/dev/sdXN}}`
```